src/modules/newtek/util/ndi.cpp

```diff
@@ -81,12 +81,7 @@ NDIlib_v3* load_library()
 
 #else
     // Try to load the library
-    void* hNDILib = dlopen(dll_path.c_str(), RTLD_LOCAL | RTLD_LAZY);
-
-    if (!hNDILib && runtime_dir) {
```

Comment:
It would be good to leave this if block as a fallback.
It is a 'standard' search path for the ndi libs, so we should try it if the system paths dont find anything

Reply:
Agreed, gonna fix this.